tardis/transport/montecarlo/estimators/dilute_blackbody_properties.py

```diff
@@ -18,11 +21,20 @@
 ).cgs.value
 
 
-class MCDiluteBlackBodyRadFieldSolver:
-    def __init__(self) -> None:
-        pass
+class MCRadiationFieldPropertiesSolver:
```

Comment:
Is this meant to be a generic solver class? The file should be renamed in that case.

Reply:
I don't know - it had that name before.

Reply:
Please rename the file to match what the class is doing